tests/robot-tests/tests/admin_and_public/bau/publish_content.robot

```diff
@@ -47,13 +47,15 @@ Add text block with link to absence glossary entry to accordion section
     user adds text block to editable accordion section    Test section    id:releaseMainContent
     ${block}=    user starts editing accordion section text block    Test section    1    id:releaseMainContent
     ${toolbar}=    get editor toolbar    ${block}
-    ${button}=    user gets button element    Add glossary entry    ${toolbar}
+    ${insert}=    user gets button element    Insert    ${toolbar}
+    user clicks element    ${insert}
+    ${button}=    user gets button element    Insert glossary link    ${toolbar}
     user clicks element    ${button}
-    user waits until modal is visible    Insert glossary link
+    ${modal}=    user waits until modal is visible    Insert glossary link
     user enters text into element    id:glossarySearch-input    abs
     user waits until page contains element    id:glossarySearch-options
     user clicks element    id:glossarySearch-option-0
-    user clicks button    Insert
+    user clicks button    Insert    ${modal}
     user waits until modal is not visible    Insert glossary link
     # Note that the way that interacting with the Glossary Search popup affects the user's focus means
```

Comment:
Take a look just below this comment.  I think we can now use the "user saves autosaving text block" keyword instead of:

```
    user clicks button    Save & close
    user waits until page finishes loading
    user waits until parent contains button    ${block}    Edit block
```

now that the focus is being returned to the editor after the featured table selection.

And the comment would be able to go too!

Reply:
Good call, I've changed it to use `user saves autosaving text block` now. I really should have read that comment 😄

Reply:
Brill thanks :slightly_smiling_face: